addons/web/views/webclient_templates.xml

```diff
@@ -369,11 +378,16 @@
                 <script type="text/javascript">
                     var odoo = {
                         csrf_token: "<t t-esc="request.csrf_token(None)"/>",
+                        debug: "<t t-esc="debug_mode"/>",
                     };
                 </script>
 
                 <t t-raw="head or ''"/>
             </head>
+            <div class="o_debug_mode text-center text-white" t-if="debug_mode" t-att-debug_mode="debug_mode">
+                <t t-set="debug_mode_help" t-value="' (%s)' % debug_mode"/>
+                Debug mode is activated<t t-esc="debug_mode in ['assets', 'tests'] and debug_mode_help"/>. <a href="?debug=" class="text-dark">Exit debug mode.</a>
```

Comment:
Why not put the condition in the definition of the variable :p ?

Reply:
Good catch, I will change that